.eslintignore

```diff
@@ -3,6 +3,7 @@ webpack.*.js
 jobs/download_cubes.js
 jobs/download_decks.js
 jobs/test_routes.js
+jobs/populate_analytics.js
```

Comment:
typically I'd agree - but since this isn't part of the 'application code' per se, I think it's fine. It does many awaits in a loop. We may just want jobs/* as our target .esltinignore